lib/resources/example_json/soarian_r4_examples_capabilitystatement.rb

```diff
@@ -0,0 +1,1140 @@
+# frozen_string_literal: true
+
+module Cerner
+  module Resources
+
+    R4_METADATA ||= {
+      "resourceType": 'CapabilityStatement',
+      "publisher": 'Cerner Corporation',
+      "date": '2022-02-07T04:02:50-05:00',
+      "url": 'https://fhir-myrecord-sc.cerner.com/r4/03416cd1-2b98-4412-95ef-920e74c5194b/metadata',
+      "name": 'Soarian Clinicals Capability Statement',
+      "title": 'Soarian Clinicals Capability Statement',
+      "status": 'active',
+      "description": 'Cerner Soarian Clinicals FHIR API',
+      "kind": 'instance',
+      "instantiates": [
+        'https://www.hl7.org/fhir/us/core/CapabilityStatement-us-core-server.html'
+      ],
+      "fhirVersion": '4.0.1',
+      "format": [
+        'json',
+        'application/json',
+        'application/fhir+json',
+        'application/json+fhir'
+
+      ],
+      "implementation": {
+        "description": 'Cerner Soarian Clinicals FHIR API',
+        "url": 'https://fhir-myrecord-sc.cerner.com/r4/2f8f5ec1-b7b8-4be5-ae27-e308284dd9c1/metadata'
```

Comment:
This URL is different than the one in Line 10 above.Please check and correct if needed

Reply:
It is same.

Reply:
@shiladityadps 
"url": 'https://fhir-myrecord-sc.cerner.com/r4/03416cd1-2b98-4412-95ef-920e74c5194b/metadata',
"url": 'https://fhir-myrecord-sc.cerner.com/r4/2f8f5ec1-b7b8-4be5-ae27-e308284dd9c1/metadata'

These are 2 different URLs

Reply:
The tenant-id is different. I have changed it.